static/js/containers/ProfilePage_test.js

```diff
@@ -1,4 +1,6 @@
 /* global SETTINGS: false */
+import '../global_init';
```

Comment:
Maybe you were running the tests through your IDE? We explicitly include this file in our mocha command in `package.json`, maybe you need to edit your test configuration to do the same for your IDE